src/sentry/static/sentry/app/views/organizationIntegrations/integrationUtil.tsx

```diff
@@ -0,0 +1,37 @@
+const pluginMapping = {
+  id: 'id',
+  name: 'name',
+};
+
+const providerMapping = {
+  id: 'key',
+  name: 'name',
+};
+
+const sentryAppMapping = {
+  id: 'slug',
+  name: 'name',
+};
+
+const getMapping = type => {
+  switch (type) {
+    case 'plugin':
+      return pluginMapping;
+    case 'sentry-app':
+      return sentryAppMapping;
+    case 'provider':
+      return providerMapping;
+    default:
+      return {};
+  }
+};
+
+export const mapped = (integration, type) => {
```

Comment:
@NisanthanNanthakumar I would prefer an API that only gets one field at a time like:
```
export const getMappedField = (integration, type, field) => {
....
}
This is because the return type of this function is an object that has an incomplete set of fields.